pyproject.toml

```diff
@@ -29,7 +29,7 @@ classifiers = [
 dynamic = ["version"]
 
 dependencies = [
-    "luna-usb~=0.2",
+    "luna-usb~=0.3",
```

Comment:
I'm not sure this extra version bump is necessary? The dep was already bumped to 0.2 in an earlier commit.

Reply:
Whoopsie, butter fingers! Fixed in: [bd12c3e](https://github.com/antoinevg/luna-soc/commit/bd12c3ea7b24793b2e7bc8e1916694d6679e1039)